README.MD

```diff
@@ -1,59 +1,66 @@
 # Goal
 
-Target is to simplify baker setup on tezos by automatizing it. 
-This script is interactive, you'll be prompted.
+This repo contains scripts & resources to ease and automate the setup of a Tezos baker.
 
-# Before running script : 
+# Prior to running the script: 
 
 ## System Requirements
 
-[x] CPU: Running with at least 2 cores is recommended
-[x] Memory: Recommended RAM for running a Tezos Node is 8GB
-[x] Disk: 100 GB SSD (SSD is highly recommended over HDD)
-[x] Unix debian based distro
-[x] Ledger Nano S or X
+- [x] CPU: Running with at least 2 cores is recommended
+- [x] Memory: Recommended RAM for running a Tezos Node is 8GB
+- [x] Disk: 100 GB SSD (SSD is highly preferred over HDD)
+- [x] A Unix debian-based distribution.
+- [x] Ledger Nano S or X
+
+## Ledger access and the plugdev group
+Ensure that the plugdev group members have read access to your ledger using the following command:
 
-## Ledger can be read by plugdev group
-Verify than member of plugdev group can read your ledger with
 ```
 getfacl /dev/bus/usb/XXX/YYY
 ```
 
-if no do :
+If they do not have access, run the following commands:
 ```
 sudo echo 'SUBSYSTEM=="usb", GROUP="plugdev", MODE="0660"' > /etc/udev/rules.d/50-myusb.rules
 sudo udevadm control --reload-rules
 sudo service udev restart
 ```
-and replug your ledger
-
-## Ledger is ready
-
- Tezos wallet is installed on your ledger
- Tezos baker app is installed on your ledger 
- Tezos baker app is launched on your ledger
- Tezos baker app watermark
-see https://www.coincashew.com/coins/overview-xtz/guide-how-to-setup-a-baker/configure-ledger-nano-s
-
-# Script flow
-Script use configuration in `script_conf.json`
-- Create baker user 
-- Download latest octez binaries
-- Setup private tezos node using systemd
-- Bootstrap tezos node from snapshot
-- Setup baker using systemd
-- Setup accuser using systemd
-- Prepare ledger wallet to bake
-- Subscribe wallet as baker on tezos chain
+After running these commands, re-plug your ledger.
+
+## Ledger Preparation
+
+Ensure the following:
+
+- The Tezos wallet is installed on your ledger.
+- The Tezos baker app is installed on your ledger.
+- The Tezos baker app is running on your ledger.
+- The Tezos baker app watermark is correctly set.
+
+Refer to [this guide](https://www.coincashew.com/coins/overview-xtz/guide-how-to-setup-a-baker/configure-ledger-nano-s) for more detailed instructions.
+
+# Script Flow
+The main entrypoint script is `main_initialize_baker.sh`.
+
+The script uses configuration details from `script_conf.json` to:
+- Create a new user account dedicated to the baker.
+- Download the latest Octez binaries.
+- Set up a private Tezos node using systemd.
+- Bootstrap the Tezos node from a snapshot.
+- Set up the baker using systemd.
+- Set up the accuser using systemd.
+- Prepare the ledger wallet for baking.
+- Register the wallet as baker on the Tezos chain.
 
 # Script configuration
 
-See script_conf.json.dist
```

Comment:
user should edit file `script_conf.json` as it is the one that is used by the script, no? @Laucans